tests/test_playbooks/usergroup.yml

```diff
@@ -0,0 +1,131 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+    - vars/usergroup.yml
+  tasks:
+  - include_tasks: tasks/user.yml
+    vars:
+      user_name: "{{ item }}"
+      user_password: "mysecret"
+      user_default_location: "{{ omit }}"
+      user_default_organization: "{{ omit }}"
+      user_locations: "{{ omit }}"
+      user_organizations: "{{ omit }}"
+    with_items: "{{ group.users }}"
+  - include_tasks: tasks/role.yml
+    vars:
+      role_name: "{{ item }}"
+      role_locations: "{{ omit }}"
+      role_organizations: "{{ omit }}"
+      role_state: present
+    with_items: "{{ group.roles }}"
+- hosts: tests
```

Comment:
```suggestion

- hosts: tests
```

Reply:
if that's not obvious from the diff (GH renders it a bit weirdly, I ask for an empty line ;))